temperature/schemas.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+from datetime import datetime
```

Comment:
Order of imports should be next:

- build-in in alphabetical order
- blank line
- third-party in alphabetical order
- blank line
- custom in alphabetical order
- two blank lines
- your code